src/content/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times.mdx

```diff
@@ -20,9 +20,15 @@ Schedule **monitor downtimes** to specify times that your synthetic [monitors](/
 * Service interruptions
 
 <Callout variant="important">
-  During synthetic monitor downtimes, your selected monitors stop running until their scheduled end time. To temporarily disable alerts without pausing your monitors, [mute them](/docs/synthetics/new-relic-synthetics/using-monitors/alerting-synthetics#mute) instead.
+  During synthetic monitor downtimes, your selected monitors stop running until their scheduled end time. To temporarily disable alerts without pausing your monitors, [mute them](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications) instead.
 </Callout>
 
+## Monitor downtime tips [#tips]
+
+A monitor downtime has a maximum duration of 24 hours. Monitor downtime windows larger than 24 hours will need to be split across multiple monitor downtime configurations.
```

Comment:
I think there's a way to clarify this... How about something like: 

"You can set how long your monitor downtime lasts. The maximum length of time for a downtime is 24 hours. If you need your downtime to last longer than 24 hours, create multiple scheduled downtimes."

feel free to tweak as you like

Reply:
Great catch. I submitted a commit that addresses this.